.bazelci/config.yaml

```diff
@@ -5,6 +5,7 @@ tasks:
     working_directory: examples
     build_targets:
       - "//cmake_android:app"
+      - "//cmake_synthetic:lib_with_duplicate_transitive_bazel_deps"
```

Comment:
Use https://github.com/bazelbuild/bazel-skylib/blob/master/rules/build_test.bzl and make it a test target?